### ISSUE TITLE
[3.x] Can't delete in Dashboard, Dialog wrong Z-index

### ISSUE THREAD

**Issue by Carreau** [ipython/ipython#8450]
1 picture worth a 1000 words.
![screen shot 2015-05-20 at 07 14 08](https://cloud.githubusercontent.com/assets/335567/7724485/f66c46d4-febf-11e4-8a33-0287514f84d9.png)

**Comment by minrk** [comment]
I've seen this, too. Reopen in incognito, does it still happen? When I ran into this, it turned out to be a cache issue, even though I don't understand how it could be.

**Comment by rgbkrk** [comment]
Yeah, I'm not sure how to reproduce this even with the 1000 words picture. :wink:

On tmpnb.org it loads fine.

**Comment by takluyver** [comment]
Bumped back to wishlist since we're not sure how to reproduce it.
